src/figuresClasses.ts

```diff
@@ -1,11 +1,96 @@
-export interface Figure {}
+// Interface
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
```suggestion
  color: Color;
```
Consider creating a type for the color